gulpfile.js

```diff
@@ -72,6 +72,7 @@ declareExtension('amp-form', '0.1', true);
 declareExtension('amp-fresh', '0.1', true);
 declareExtension('amp-fx-flying-carpet', '0.1', true);
 declareExtension('amp-gfycat', '0.1', false);
+declareExtension('amp-hulu', '0.1', false, 'NO_TYPE_CHECK');
```

Comment:
Please remove `NO_TYPE_CHECK` and allow the component to be type checked. Please run `gulp check-types` locally to ensure it passes type checking.